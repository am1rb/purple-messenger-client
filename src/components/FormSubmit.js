// @flow

import React from 'react'

import FormButton from './FormButton'

type Props = {
	type?: string,
	label?: string,
	autoFocus?: boolean,
};

export default class FormSubmit extends React.PureComponent<Props>
{
	render()
	{
		const {type, ...props} = this.props

		return (
			<FormButton
				type="submit"
				{...props}
			/>
		)
	}
}
