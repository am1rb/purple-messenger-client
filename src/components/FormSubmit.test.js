import React from 'react'
import renderer from 'react-test-renderer'

import FormSubmit from './FormSubmit'


describe("<FormSubmit /> component tests", () => {

	it("Should not regress", () => {

		const component = renderer.create(
			<FormSubmit label="Button" />
		)

		expect(component.toJSON()).toMatchSnapshot()

	})

})