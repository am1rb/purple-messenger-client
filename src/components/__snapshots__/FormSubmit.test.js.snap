// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FormSubmit /> component tests Should not regress 1`] = `
<button
  type="submit"
>
  Button
</button>
`;
